.mvn

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I am never sure which is generated and fine to include and which is not. 
this seems like polyglot mvn's generated temporary piece (due pom.rb)?

Reply:
From my understanding the `.mvn` folder and extensions.xml is required for maven to recognize that we are using polyglot maven (pom.rb) and not regular maven (pom.xml), the change I proposed just bumps the version, for completeness we generate pom.xml but it is not actually used.
https://github.com/takari/polyglot-maven-examples/blob/master/ruby/.mvn/extensions.xml

Reply:
@kares It seem things are not as straightforward as I thought, as the latest version of maven does not seem to require the `.mvn` folder, and `pom.rb` is sufficient. But for the moment we are probably safer following takari master.